index.html

```diff
@@ -13,6 +13,66 @@
 </head>
 
 <body>
+    <div class="educatinal-details">
+        <div class="card">
+            <h5 class="card-header">Educational Details</h5>
+            <div class="card-body">
+                <div class="detail">
+                    <div class="row">
+                        <div class="col-6">
+                            <h6>Master of Technology in Computer Science </h6>
+                        </div>
+                        <div class="col-6">
+                           
+                            <label class="fw-bolder">  May 2024</label>
+
+                        </div>
+                        <div class="col-12 mb-2">
+                           <strong class="text-capitalize"> pandit shivdatta shastri education foundation's asma institute of management Shivane Pune</strong>
+                        </div>
+                               <hr>
+                        <div class="col-6">
+                            <h6>Bachelore of Technology in Computer Science </h6>
```

Comment:
If you want you can highlight course without highlighting college name.

Reply:
resolved education page  structure